core/templates/dev/head/app.js

```diff
@@ -425,6 +425,7 @@ oppia.factory('siteAnalyticsService', ['$window', function($window) {
   // developers.google.com/analytics/devguides/collection/analyticsjs/events
   var _sendEventToGoogleAnalytics = function(
       eventCategory, eventAction, eventLabel) {
+    console.log(eventCategory + ' ' + eventLabel);
```

Comment:
drop this line? Only needed during development, but doesn't need to be submitted.